src/utils/helper.js

```diff
@@ -0,0 +1,66 @@
+import { getToken, verifyToken } from './authentication';
+
+export const returnDate = (newdate, oldDate) => {
+  if (newdate) {
+    const date = new Date(newdate);
+    return date.toLocaleDateString();
+  }
+  return oldDate;
+};
+
+export const confirmUser = (req, res) => {
+  const { decodedPayload, success } = verifyToken(getToken(req));
+  if (!success) {
+    return res.status(401).json({
```

Comment:
Authorization has already been handled. To secure a route, use the `checkToken` middleware  in `src/middlewares/authorization.js` file. Here is a sample:
```
...
import { checkToken } from '../../middlewares/authorization';
...

router.patch('/user/:id', checkToken, Profile.updateProfile);
router.get('/user/:id', checkToken, Profile.getProfile);
```

> It works like magic :-)

Hence, you may not  need to create a `getToken` function.